index.js

```diff
@@ -77,7 +79,12 @@ function getInfo (cpr) {
 // list all valid cpr numbers for a date, using modulo-11 and century check
 function validForDate (date) {
   var year = date.getFullYear()
-  var dateString = moment(date).format('DDMMYY')
+
+  var formattedDate = ('0' + date.getDate()).slice(-2);
+  var formattedMonth = ('0' + (date.getMonth() + 1)).slice(-2);
+  var formattedYear = date.getFullYear().toString().substr(2,2);
```

Comment:
Is there a reason for using `substr` and not `slice` here?

Reply:
no reason :) a late hour + copy/paste